README.rst

```diff
@@ -62,14 +62,15 @@ The following is the output from ``ansible-lint --help``, providing an overview
 
 .. code-block:: bash
 
-    Usage: ansible-lint playbook.yml|roledirectory ...
+    Usage: ansible-lint [options] [playbook.yml [playbook2 ...]]
```

Comment:
we shouldn't remove `roledirectory` here

Reply:
I didn't really remove it so much as use what was in the output of the actual utility. I put it back in, but I'd think it would be in the actual --help output too, not just appended in the docs.

Reply:
Ah, I didn't realize our `--help` was missing that, I will add it there. Thanks!